demo/index.js

```diff
@@ -36,55 +36,59 @@ $(function() {
 
         platform = sdk.platform();
 
-        // TODO: Improve later to support international phone number country codes better
-        if (login) {
-            login = (login.match(/^[\+1]/)) ? login : '1' + login;
-            login = login.replace(/\W/g, '')
-        }
+        var loginUrl = platform.loginUrl({implicit: false, redirectUri: redirectUri});
 
-        platform
-            .login({
-                username: login,
-                extension: ext || null,
-                password: password
-            })
-            .then(function() {
+        this.loginPopupUri  = function(loginUrl, redirectUri) {
 
-                logLevel = ll;
-                username = login;
+            platform
+                .loginWindow({url: loginUrl, redirectUri: redirectUri}) // this method also allows to supply more options to control window position
+                .then(function (loginOptions){
+                    loginOptions.redirectUri = redirectUri;
+                    return platform.login(loginOptions);
+                })
```

Comment:
This is not needed if you provide redirect URI second time here if it was provided to SDK constructor, so the whole thing converts to:

```js
.then(function (platform.login.bind(platform))
```